reducers/chatsReducer.js

```diff
@@ -24,17 +25,19 @@ function chatsReducer(state = chatsInitialState, action) {
                 chats: state.chats.concat(payload)
             };
         case types.CREATE_USER_IN_CHAT:
-            const { cid, userId, displayName } = payload;
+            const { id, display_name: displayName, cid } = payload;
+            const { chats } = state;
```

Comment:
chats doesn't look at all like the format that you proposed earlier and i mentioned in the pr https://github.com/emote-chat/server/pull/21, or am i losing it?

Reply:
i didn't touch it since i'm assuming this format was worked with in other views already that it could break things unless changed/checked for everywhere, so maybe we can revisit the last couple weeks when we have some breathing room!

Reply:
yeah i dont think it does... we can check it later!

Reply:
cool, i added all - i think - of the stuff that we've discussed the past week or so we might want to revisit in the last couple weeks we have.